exercises/practice/leap/.approaches/flow/content.md

```diff
@@ -0,0 +1,97 @@
+# Condition flow structures
```

Comment:
They're called "control flow structures" in the concept documents, so let's be consistent:
```suggestion
# Control flow structures
```